app/assets/stylesheets/application.css.scss

```diff
@@ -16,15 +16,33 @@
 
  */
 
+body {
+  background-color: white;
+}
+
+.name {
+  padding-top: 7px;
+  padding-right: 10px;
+}
+
 .review_body {
   max-width: 500px;
   word-wrap: break-word;
 }
 
+.top-bar {
+  background-color: #7EBAC2;
```

Comment:
Color literals like `#7EBAC2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#7EBAC2` should be written as `#7ebac2`